clang/tools/c-index-test/c-index-test.c

```diff
@@ -5199,13 +5199,13 @@ static void flush_atexit(void) {
 int main(int argc, const char **argv) {
   thread_info client_data;
 
-#ifdef __MVS__
+  // On z/OS we need to enable auto conversion
   if (enablezOSAutoConversion(fileno(stdout)) == -1)
```

Comment:
I suggest defining `enableAutoConversion()` as an inline function in AutoConvert.h that always returns true for non z/OS platforms and for z/OS it calls this function.  That will allow us to:
- not add unneeded function calls in the final code.
- keep the AutoConvert.cpp strictly for z/OS.
This would go for all functions in AutoConvert.h.

Reply:
done, thanks!